### ISSUE TITLE
Can SVF be applied on C++ bitCode?

### ISSUE THREAD

**Issue by qzhou2017** [SVF-tools/SVF#226]
I have a question:

Can SVF be applied on C++ bitCode?

Because currently I need to analyze C++ project files.

**Comment by yuleisui** [comment]
Yes, SVF analyzes LLVM bcs. You will need to compiling your C++ programs into bcs using clang++.
